test/client-test.js

```diff
@@ -27,17 +27,21 @@ describe("client", function() {
       sinon.assert.calledWithNew(window.EventSource);
       sinon.assert.calledWith(window.EventSource, '/__webpack_hmr');
     });
-    it("should trigger webpack on successful builds", function() {
-      var eventSource = window.EventSource.lastCall.returnValue;
-      eventSource.onmessage(makeMessage({
-        action: 'built',
-        time: 100,
-        hash: 'deadbeeffeddad',
-        errors: [],
-        warnings: [],
-        modules: []
-      }));
-      sinon.assert.calledOnce(processUpdate);
+    it("should trigger webpack on successful builds / syncs", function() {
+      const actions = ['built', 'sync'];
+      actions.forEach(function(action) {
```

Comment:
Could you move this loop outside the `it()`, or manually repeat the test so that it's two distinct cases?